sktime/classification/dictionary_based/_cboss.py

```diff
@@ -221,7 +221,8 @@ def _fit(self, X, y):
         if self.typed_dict != "deprecated":
             warnings.warn(
                 "``typed_dict`` was deprecated in version 0.13.3 and "
-                "will be removed in 0.15."
+                "will be removed in 0.15.",
```

Comment:
hm, this looks like a missed deprecation - can you kindly open an issue so we track this and someone checks whether this should be removed?

Reply:
In which category should this issue be opened? This seems like a development enhancement `[ENH]`.

Reply:
I think it's "maintenance", [MNT], since a developer or release manager should have deprecated this, but probably the deprecation note was not attached correctly, or it was not on the release board, etc (someone needs to check what happened and what needs doing)

Reply:
Created #4313